clang/include/clang/Sema/ExternalSemaSource.h

```diff
@@ -181,6 +181,9 @@ class ExternalSemaSource : public ExternalASTSource {
                  SmallVectorImpl<std::pair<ValueDecl *,
                                            SourceLocation> > &Pending) {}
 
+  virtual void ReadPendingOfInstantiationsForConstexprEntity(
```

Comment:
The "Of" here seems out of place; maybe remove it? You also have an inconsistent "Of" vs "For" after "Instantiations". Would be good to use a consistent spelling.